### ISSUE TITLE
Repository not found

### ISSUE THREAD

**Issue by amplicade** [project-ubin/ubin-fabric#1]
Hi,

when trying to clone `git clone http://gitlabnx01.southeastasia.cloudapp.azure.com/dev-fabric/ubin-fabric.git` it asks for credentials. I have registered with Gitlab but the repository is not in my list so I can't clone it. Is this by purpose?

Regards,
Tac

**Comment by cchew** [comment]
Hey Tac, does 'git clone https://github.com/project-ubin/ubin-fabric.git' work for you?

**Comment by amplicade** [comment]
Yes it does. I'm following the `README.md` and in step 1 it asks to clone the repository in gitlabnx01. Can I skip this step if I cloned the `https://github.com/project-ubin/ubin-fabric.git'` ?

**Comment by cchew** [comment]
Yes, you can skip the clone step from the instructions if you have already cloned the GitHub repo.
